compiler/rustc_parse/src/lexer/unescape_error_reporting.rs

```diff
@@ -158,6 +158,13 @@ pub(crate) fn emit_unescape_error(
                     "this is an isolated carriage return; consider checking your editor and \
                      version control settings",
                 );
+            } else if looks_like_quote(c) {
+                diag.help(format!(
+                    "{ec} is not an ascii quote, \
+                             but may look like one in some fonts.\n\
+                             consider writing it in its \
+                             escaped form for clarity."
```

Comment:
the spacing is a bit awkward here. Can we use a semicolon to turn this into one line?

```suggestion
                    "{ec} is not an ascii quote but may look like one in some fonts; consider escaping it to avoid confusion"
```

Reply:
i'm pretty sure that would put it over the 100 char line length limit.

i'm also unsure if you have a problem with the formatting of the *output* or the *code* (the code is 4 lines, but the actual output is only 2)

Reply:
The output -- we don't typically have multi-line diagnostics (unless formatting a list or something), and we try to avoid periods in diagnostic outputs as a matter of style.

Yeah, you'll need to re-`\` the string literal.